### PR TITLE
Edit firewall openings to clarify replicated proxy service and private registry

### PR DESCRIPTION
**Pull request by paigecalvert**
https://deploy-preview-1040--replicated-docs.netlify.app/enterprise/installing-general-requirements#firewall-openings-for-online-installations

Story: https://app.shortcut.com/replicated/story/72874/edit-firewall-requirements-in-installation-reqs-topic

### CODE REVIEW

File: docs/enterprise/installing-general-requirements.mdx [hunk 1]
```diff
@@ -274,9 +274,14 @@ No outbound internet access is required for air gapped installations.
...
 | k8s.kurl.sh          | Not Required                  | Required                      | Kubernetes cluster installation scripts and artifacts are served from [kurl.sh](https://kurl.sh). An application identifier is sent in a URL path, and bash scripts and binary executables are served from kurl.sh. This domain is owned by Replicated, Inc., which is headquartered in Los Angeles, CA.                                                     |
 | amazonaws.com        | Not Required                  | Required                      | tar.gz packages are downloaded from Amazon S3 during embedded cluster installations. For information about dynamically scraping the IP ranges to allowlist for accessing these packages, see [AWS IP address ranges](https://docs.aws.amazon.com/general/latest/gr/aws-ip-ranges.html#aws-ip-download) in the AWS documentation.                                                         |
+
+&#42; Required only if the application uses the Replicated proxy service. Contact your application vendor for more information.
+
+&#42;&#42; Required only if the application uses the Replicated private registry service. Contact your application vendor for more information.
```
Comment (ajp-io) [inline docs/enterprise/installing-general-requirements.mdx:287]: Nbd but it looks like elsewhere we stop after "registry" and don't also say "service".
```suggestion
&#42;&#42; Required only if the application uses the Replicated private registry. Contact your application vendor for more information.
```

File: docs/enterprise/installing-general-requirements.mdx [hunk 2]
```diff
@@ -274,9 +274,14 @@ No outbound internet access is required for air gapped installations.
...
 | replicated.app       | Required                      | Required                      | Upstream application YAML and metadata is pulled from replicated.app. The current running version of the application (if any) will be sent, in addition to a license ID and an application IDs are sent to replicated.app to authenticate and receive these YAML files. This domain is owned by Replicated, Inc., which is headquartered in Los Angeles, CA. |
+| proxy.replicated.com | Required&#42;                 | Required&#42;                 | Upstream Docker images are proxied via proxy.replicated.com. The on-prem docker client uses a license ID to authenticate to proxy.replicated.com. This domain is owned by Replicated, Inc which is headquartered in Los Angeles, CA.                                                                                                                       |
```
Comment (jonquil2002) [inline docs/enterprise/installing-general-requirements.mdx:278]: Suggest changing "via" to "using" or something else, per our Word List.

**Review by jonquil2002 — DISMISSED** [review]
lgtm - one minor word change suggested for Word List compliance.

**Comment by paigecalvert** [comment]
@ajp-io, this is about ready to merge, but wanted to check with you first to make sure you had finished your review.

File: docs/enterprise/installing-general-requirements.mdx [hunk 3]
```diff
@@ -274,9 +274,14 @@ No outbound internet access is required for air gapped installations.
...
 | replicated.app       | Required                      | Required                      | Upstream application YAML and metadata is pulled from replicated.app. The current running version of the application (if any) will be sent, in addition to a license ID and an application IDs are sent to replicated.app to authenticate and receive these YAML files. This domain is owned by Replicated, Inc., which is headquartered in Los Angeles, CA. |
+| proxy.replicated.com | Required&#42;                 | Required&#42;                 | Upstream Docker images are proxied through proxy.replicated.com. The on-prem docker client uses a license ID to authenticate to proxy.replicated.com. This domain is owned by Replicated, Inc which is headquartered in Los Angeles, CA.                                                                                                                       |
```
Comment (ajp-io) [inline docs/enterprise/installing-general-requirements.mdx:278]: Should clarify that these are private images. Public images aren't proxied.

File: docs/enterprise/installing-general-requirements.mdx [hunk 4]
```diff
@@ -274,9 +274,14 @@ No outbound internet access is required for air gapped installations.
...
+| registry.replicated.com | Required&#42;&#42;         | Required&#42;&#42;            | Some applications host private images in the Replicated private registry at this domain. This domain is owned by Replicated, Inc which is headquartered in Los Angeles, CA.
```
Comment (ajp-io) [inline docs/enterprise/installing-general-requirements.mdx:279]: I think license ID is used for this too, so we could copy that wording from proxy above.

File: docs/enterprise/installing-general-requirements.mdx (hunk 3 above)
Comment (ajp-io) [reply docs/enterprise/installing-general-requirements.mdx:278]: Maybe this?
```suggestion
| proxy.replicated.com | Required&#42;                 | Required&#42;                 | Private Docker images are proxied through proxy.replicated.com. The on-prem docker client uses a license ID to authenticate to proxy.replicated.com. This domain is owned by Replicated, Inc which is headquartered in Los Angeles, CA.                                                                                                                       |
```

**Comment by paigecalvert** [comment]
Merging

**Pull request merged**